src/NuGetPackageVerifier/CompositeRules/SigningVerificationCompositeRule.cs

```diff
@@ -15,6 +15,7 @@ public class SigningVerificationCompositeRule : IPackageVerifierRule
             new AuthenticodeSigningRule(),
             new PowerShellScriptIsSignedRule(),
             new RequiredNuSpecInfoRule(),
+            new DefaultCompositeRule(),
```

Comment:
Ah sure, that's fine. It should be quick.

Which assemblies were missing metadata/descriptions?